### ISSUE TITLE
migrate to Vite (brighter future than Create React App and Webpack)

### ISSUE THREAD

**Issue by pdurbin** [IQSS/dataverse-frontend#10]
This issue is a follow on to the conversation at https://github.com/IQSS/dataverse-frontend/pull/4#issuecomment-1443390932

In short, the skeleton app in PR #4 is based on Create React App, but there is a [long writeup by one of its creators](https://github.com/reactjs/reactjs.org/pull/5487#issuecomment-1409720741) that explains some potential problems with it:

- "As the years passed, Create React App has stagnated."
- "We could suggest people to migrate from Create React App to something like Vite."

In short, Vite seems to have a brighter future than Create React App. I think Vite would be a stronger foundation for us. This issue is about trying to migrate to Vite.

**Comment by mreekie** [comment]
@MellyGray @pdurbin Would it be possible to add a blurb that puts a bit of scope around this?
Also - can you take a shot at a size for this?
Base the size on what's left, since we are starting the new sprint today.

**Comment by rtreacy** [comment]
@mreekie What does it mean to say that we're not answering an investigative discussion? It needs to be investigated whether moving to Vite will interfere with using certain components/frameworks that are available in React, but are not yet usable with Vite

**Comment by mreekie** [comment]
> @mreekie What does it mean to say that we're not answering an investigative discussion? It needs to be investigated whether moving to Vite will interfere with using certain components/frameworks that are available in React, but are not yet usable with Vite

@rtreacy hopefully I clarified it.  That was awkwardly worded.

**Comment by pdurbin** [comment]
During standup @MellyGray indicated she has made a lot of progress already (🎉 🎉 🎉 ) so an 80 might be high.

**Comment by MellyGray** [comment]
@pdurbin Yes, I already did a lot of progress today, I lowered the size to 33, I still need to migrate Storybook and fix the GitHub workflows. Does 33 sounds good to you?
